systemd/ceph-exporter.target

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=ceph target allowing to start/stop all ceph-exporter@.service instances at once
+PartOf=ceph.target
+After=ceph-mon.target
```

Comment:
why shall this target start after `ceph-mon`?

Reply:
I found the following PR, and I thought I should follow the changes in this PR.
https://github.com/ceph/ceph/pull/36776

> ceph-*.target: Ensure startup after ceph-mon.target and before remote-fs-pre.target

Reply:
i don't think it's relevant. as ceph-exporter does not depends on monitor. strictly speaking it does not depend on any services. but "ceph.target" could depend on it. as before it is up and running, Prometheus would not be able to pull the metrics from it, if it is configured to do so.

Reply:
I removed this line of code.